n_t/msmc.py

```diff
@@ -22,14 +22,16 @@ def write_msmc_file(path):
     for var in ts.variants():
         cur = int(var.site.position)
         if cur > prev:
-            string = chrom+'\t'+str(cur)+'\t'+str(cur-prev)+'\t'+''.join(map(str,var.genotypes))+"\n"
-            fi.write(string)
+            #string = chrom+'\t'+str(cur)+'\t'+ str(cur-prev)
+            #string_2 = '\t'+''.join(map(str,var.genotypes))+"\n"
+            geno = ''.join(map(str,var.genotypes))
+            fi.write(f"{chrom}\t{cur}\t{cur-prev}\t{geno}\n")
```

Comment:
@jgallowa07 msmc can only deal with small samples, and smc++ and stairwayplot both can use larger sample sizes. I think we want to subsample our original tree sequences, choosing some specified number of individuals at random, and use that subsample for msmc. Can you please adjust this function to do that?

Reply:
totally, np.